include/rosdiscover-clang/Stmt/ApiCall.h

```diff
@@ -78,12 +78,14 @@ class Subscriber : public SymbolicRosApiCall {
     return {
       {"kind", "subscribes-to"},
       {"name", getName()->toJson()},
-      {"format", format}
+      {"format", format},
+      {"callback", callback->toJson()}
     };
   }
 
 private:
   std::string const format;
+  std::unique_ptr<SymbolicStmt> callback;
```

Comment:
not sure where my earlier comment went. SymbolicStmt seems too permissive (i.e., it doesn't make sense for any arbitrary statement to be a callback)